src/System.Console/src/System.Console.csproj

```diff
@@ -49,6 +46,15 @@
   <!-- Windows : Win32 -->
   <ItemGroup Condition="'$(TargetsWindows)' == 'true' And '$(TargetGroup)' == ''">
     <Compile Include="System\ConsolePal.Windows.cs" />
+    <Compile Include="$(CommonPath)\System\Text\OSEncoding.Windows.cs">
+      <Link>Common\System\Text\OSEncoding.Windows.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\System\Text\EncodingTable.cs">
+      <Link>Common\System\Text\EncodingTable.cs</Link>
```

Comment:
We should not be bundling multiple copies of these tables just to return the proper encoding name that are very unlikely to be ever called.

Instead, the name properties should be implemented by getting the global encoding for the given codepage via `Encoding.GetEncoding` and forwarding the request to it. If you do not register the provider, you will get an exception telling you to register the full encoding provider to get the encoding name - you will be only able to get the names of these encodings if you register the full encoding provider.